research/combine_aq_airports_data.py

```diff
@@ -0,0 +1,69 @@
+"""
+Program to inject air quality data into the airports list from FlightRadar24
+
+IMPORTANT:
+- This file must be run AFTER the "open_weather_search.py" file
+- This file requires a configuration file to run successfully
+- The configuration file must be in the "research" directory and be named "config.yaml"
+- The configuration file must at least contain the following:
+
+file_paths:
+  airports_load: str (path the JSON file containing information about airports, see below)
+  air_quality_load: str (path to air quality measurements obtained from "open_weather_search.py" file)
+  airports_augmented_dump: str (path to airport data with air quality measurements now integrated)
+
+airports_load file structure:
+{
+    airports: [
+        {
+            “altitude”: int
+            “country”: str
+            “iata”: str
+            “icao”: str
+            “name”: str
+        },
+        {},
+        ...
+    ]
+}
+"""
```

Comment:
This is quite confusing to me

Reply:
if the airports.json data is not inside the repo, shouldn't i tell the user what structure that data file should have so that the script runs properly?